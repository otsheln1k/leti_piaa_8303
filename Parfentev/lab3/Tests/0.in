5
a
d
a c 1
a b 1
b d 1
c d 1
b c 1
