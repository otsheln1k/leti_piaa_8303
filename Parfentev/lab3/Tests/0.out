2
a b 1
a c 1
b c 0
b d 1
c d 1
